tests/publisher/tests.py

```diff
@@ -52,14 +49,14 @@ def test_hub_declaration(self):
         response = self.client.get(reverse('feed'))
         hub_declaration = response.content.decode('utf-8').split(
             '</updated>', 1)[1].split('<entry>', 1)[0]
-        self.assertEqual(len(hub_declaration), 53)
+        self.assertEqual(len(hub_declaration), 47)
```

Comment:
Why is the length of messages being changed here? Is it because of Python3 unicode str?

Reply:
The only difference is that by updating the Django version the generated code changes slightly. For example int his point:
```diff
- <link href="http://hub.example.com" rel="hub"></link>
+ <link href="http://hub.example.com" rel="hub"/>
```

Reply:
Oh, I see 👀 

Thanks, overall look good to me!